x-pack/plugins/snapshot_restore/server/lib/booleanize_settings.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { Repository } from '../../common/types';
+
+/**
+ * Utility to coerce ES boolean strings to actual booleans
+ */
+export const booleanizeSettings = (repository: Repository) => {
```

Comment:
Same comment as above. I'd give it a generic name so it could be used and reused at will 😊

Reply:
Maybe as a compromise we can follow the rule of threes, and wait for multiple use cases to arise before generalizing?

Reply:
So here I was just saying to give it a generic _name_. By now we should have a common library of all those generic utils functions instead of re-writing them over and over.